packages/js-moi-logic/__tests__/logic.test.ts

```diff
@@ -1,13 +1,12 @@
-import erc20 from "../manifests/erc20.json";
-import {JsonRpcProvider} from "js-moi-providers";
-import {LogicFactory} from "../src/logic-factory";
-import { getLogicDriver } from "../src/logic-driver";
-import { initializeWallet } from "./utils/utils";
+import { JsonRpcProvider } from "js-moi-providers";
 import { Signer } from "js-moi-signer";
 import { Wallet } from "js-moi-wallet";
+import todo from "../manifests/todo.json";
```

Comment:
replace the todo logic with erc20 logic.

Reply:
a148b25d7be6dd2e55ef9f4502f56ebc49018142